configure

```diff
@@ -554,13 +556,16 @@ else
 fi
 
 check_define() {
-cat > $TMPC <<EOF
-#if !defined($1)
-#error $1 not defined
-#endif
-int main(void) { return 0; }
-EOF
-  compile_object
+  if [ "$1" == "__linux__" ]; then
+    echo "WORKAROUND"
+    return 0
+  fi
+  if [ "$1" == "__x86_64__" ]; then
```

Comment:
Can you change these comparisons to `=`? This file invokes posix sh, and technically test for strings is only one `=`. Dash, the ubuntu default, does not like the double.